sourceme.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh
 # Convenience script for setting variables for in-tree use of hammer.
+echo "Installing dependency"
+python3 -m pip install --user typing six enum34 python-jsonschema-objects pyaml pytest
 if [ -z "$HAMMER_HOME" ]; then
     >&2 echo "Must set HAMMER_HOME to root. Try export HAMMER_HOME="\$PWD" for the current dir."
 else
+echo "Setting PYTHONPATH and PATH"
```

Comment:
Could you avoid these `echo`es in the script?

Reply:
OK!